### PR TITLE
Font and navbar

### PR DESCRIPTION
**Pull request by jaynopponep**
- fixed fonting issues
- fixed navbar to look up to standard
- fixed scrolling issue of navbar not disappearing
- fixed color changing back and forth in the eboard section
- fixed 'orange box' glitch at the end of eboard scroll
- removed out of place lamp component 
- changed padding of the 'home' component to decrease empty space between home & eboard component

### CODE REVIEW

File: src/components/images/BYTE.png
Comment (LordFarquaadtheCreator) [inline src/components/images/BYTE.png:1]: ur gonna have to remove this, ill host the image on our s3 bucket. gimme a min

**Review by LordFarquaadtheCreator — CHANGES_REQUESTED** [review]
use this link for the image instead:
https://byte-media.s3.us-east-2.amazonaws.com/byte-group.jpg

File: src/components/images/BYTE.png
Comment (jaynopponep) [reply src/components/images/BYTE.png:1]: used the link that you gave me for source & removed the local jpg image

**Comment by jaynopponep** [comment]
deleted png image, used source link you gave me instead

**Review by LordFarquaadtheCreator — APPROVED** [review]

**Pull request merged**